compiler/src/iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -1113,11 +1050,86 @@ void DispatchLinalgOnTensorsPass::runOnOperation() {
   }
 }
 
+/// Create dispatch.region Ops based on a fusion heuristic.
+void FormDispatchRegionsPass::runOnOperation() {
```

Comment:
IIUC this is a move of the `createDispatchRegion` method. Can be we keep the original method and just call it from here... will reduce the code diff. And also the move here dropped the `LLVM_DEBUG` usage that I have found extremely useful for debugging large models.